optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/TestGenValueFactTest.java

```diff
@@ -40,7 +38,7 @@ public void initialization() {
 
         StringBuilder sb = new StringBuilder(100);
         fact.printInitialization(sb);
-        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();\n", sb.toString());
+        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();" + System.lineSeparator(), sb.toString());
```

Comment:
I think we need to use \n under all circumstances (including windows) because we already enforce that on the drools/optaplanner repo's (see README in build-bootstrap) and through .gitattributes (not sure if that's in the drools repo yet, but it should be)